### PR TITLE
Import `loop_in_thread` fixture in tests

### PR DESCRIPTION
**Pull request by jrbourbeau**
Following https://github.com/dask/distributed/pull/6680 we started seeing failures like 

```python
 ____________ ERROR at setup of test_blockwise_fusion_after_compute _____________
[gw2] linux -- Python 3.10.5 /usr/share/miniconda3/envs/test-environment/bin/python
file /home/runner/work/dask/dask/dask/tests/test_distributed.py, line 416
  def test_blockwise_fusion_after_compute(c):
file /usr/share/miniconda3/envs/test-environment/lib/python3.10/site-packages/distributed/utils_test.py, line 596
  @pytest.fixture
  def client(loop, cluster_fixture):
file /usr/share/miniconda3/envs/test-environment/lib/python3.10/site-packages/distributed/utils_test.py, line 145
  @pytest.fixture
  def loop(loop_in_thread):
E       fixture 'loop_in_thread' not found
>       available fixtures: c, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cleanup, cluster_fixture, cov, doctest_namespace, loop, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, shuffle_method, testrun_uid, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, worker_id
>       use 'pytest --fixtures [testpath]' for help on them.

/usr/share/miniconda3/envs/test-environment/lib/python3.10/site-packages/distributed/utils_test.py:145
```

in [CI builds](https://github.com/pavithraes/dask/runs/7619890993?check_suite_focus=true). 

xref https://github.com/dask/distributed/issues/6806 for a related conversation around what testing utilities in `distributed` should be considered public. This PR just imports the missing pytest fixture as a quickfix to unblock CI

cc @hendrikmakait for visibility

### CODE REVIEW

File: dask/tests/test_distributed.py [hunk 1]
```diff
@@ -21,6 +27,7 @@
 from dask.distributed import futures_of, wait
 from dask.highlevelgraph import HighLevelGraph, MaterializedLayer
 from dask.utils import get_named_args, tmpdir, tmpfile
+from dask.utils_test import inc
```
Comment (jrbourbeau) [inline dask/tests/test_distributed.py:30]: Unrelated change. We already have an `inc` function in the `dask.utils_test` so no need to import it from `distributed.utils_rest`

**Review by ian-r-rose — APPROVED** [review]
Unfortunate, but let's get CI working. Thanks @jrbourbeau

**Comment by jrbourbeau** [comment]
Note the two `scipy`-related test failure have been resolved in https://github.com/dask/dask/pull/9336. As the changes here unblock CI and are (in my opinion) pretty uncontroversial, I'll plan to merge this in after CI finishes

**Comment by jrbourbeau** [comment]
Thanks for the review @ian-r-rose (also, welcome back!)
